pkg/ext-proc/backend/provider.go

```diff
@@ -2,23 +2,25 @@ package backend
 
 import (
 	"context"
-	"fmt"
 	"sync"
 	"time"
 
-	"go.uber.org/multierr"
 	klog "k8s.io/klog/v2"
 )
 
 const (
-	fetchMetricsTimeout = 5 * time.Second
+	// TODO: make it configurable. One idea is to provide a configuration singleton
+	// and put fields like refreshMetricsInterval in it. So far, we have to pass these
+	// fields across several layers.
+	metricsValidityPeriod = 5 * time.Second
```

Comment:
Pls also document that currently stale metrics have no effect except the debug logging, and can you create a issue to follow up on dealing with stale metrics?

Reply:
I will create an issue for this once the PR is merged. Where should I doc the behavior?

Reply:
You can create an issue and link the issue here in the comment as documentation